out/notebooks/controller.ts

```diff
@@ -11,19 +11,17 @@ import {
 import { splunkMessagesToOutputItems } from './utils';
 
 export class SplunkController {
```

Comment:
All the changes to this file were simply to make it easier to extend/inherit this class to reuse most of the code in here.